RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.h

```diff
@@ -58,6 +59,12 @@ class HGCalRecHitWorkerSimple : public HGCalRecHitWorkerBaseClass {
   std::vector<float> weights_, weightsNose_;
   std::unique_ptr<HGCalRecHitSimpleAlgo> rechitMaker_;
   std::unique_ptr<hgcal::RecHitTools> tools_;
+
+  double minValSiP_;
```

Comment:
Is there any reason why these parameters are class members? They are only used as local variables inside the constructor.
Apparently only ```timeEstimatorSi_``` needs to be a class member